src/states_screens/race_gui.cpp

```diff
@@ -1158,13 +1185,30 @@ void RaceGUI::drawLap(const AbstractKart* kart,
     // don't display 'lap 0/..' at the start of a race
     if (lap < 0 ) return;
 
+    // Display lap flag
+    irr::video::ITexture *lap_flag= irr_driver->getTexture(FileManager::GUI_ICON,
```

Comment:
Calling getTexture repeatedly on each render is a bad practice, this will do a string lookup each time. BEst keep a reference to the texture